omnibus/resources/agent/msi/localbuild/parameters.wxi

```diff
@@ -8,7 +8,7 @@
   <?define BinFiles="c:\omnibus-ruby\src\datadog-agent\src\github.com\DataDog\datadog-agent\bin\agent" ?>
   <?define EtcFiles="C:/omnibus-ruby/src\etc\datadog-agent" ?>
   <?define ResourcesDir="c:\dev\go\src\github.com\datadog\datadog-agent\omnibus\resources\agent\msi"?>
-  <?define IncludePython2= "true" ?>
+  <?define IncludePython2= "false" ?>
```

Comment:
Unintended change?

Reply:
No, it was intended.  And it doesn't affect the actual build.  this file is for generating installs for testing without doing a whole omnibus build.  It's developer use only.